frontend/public/components/factory/details.tsx

```diff
@@ -55,6 +58,7 @@ export type DetailsPageProps = {
   pages?: Page[];
   pagesFor?: (obj: K8sResourceKind) => Page[];
   kind: K8sResourceKindReference;
+  kindObj?: K8sKind;
```

Comment:
Do we need both `kind` and `kindObj`?

It looks like we always have a `kindObj` if I'm reading the code below correctly.

Reply:
Follow on to do:  update to only pass `kindObj`.